cli/flox/src/commands/mod.rs

```diff
@@ -67,7 +67,7 @@ static FLOX_WELCOME_MESSAGE: Lazy<String> = Lazy::new(|| {
 });
 
 const ADDITIONAL_COMMANDS: &str = indoc! {"
-    update, upgrade, config, wipe-history, history, auth
+    update, upgrade, config, auth
```

Comment:
question: did you mean to add containerize?

Reply:
No, we're apparently not advertising that command yet

Reply:
Then should it be in the man page under additional commands?

Reply:
Or at least should we add `(experimental)`?

Reply:
I just removed that as well.